pyuploadtool/changelog/types.py

```diff
@@ -0,0 +1,23 @@
+import os
+from enum import Enum
+
+
+class ChangelogType(Enum):
+    # default
+    STANDARD = 0
+
+    # follows the Conventional Commit Spec
+    CONVENTIONAL = 1
+
+    @staticmethod
+    def from_environment():
+        _type = os.getenv("CHANGELOG_TYPE")
```

Comment:
Same question: why are you using an underscore prefix?

Reply:
I am not sure why :man_facepalming: :shrug: lol.